src/custom/utils/trade.ts

```diff
@@ -190,25 +191,22 @@ export async function signAndPostOrder(params: PostOrderParams): Promise<AddUnse
   let signature: string | undefined
   if (allowsOffchainSigning) {
     const signedOrderInfo = await signOrder(unsignedOrder, chainId, signer)
-    signingScheme = signedOrderInfo.signingScheme
+    signingScheme =
+      signedOrderInfo.signingScheme === EcdsaSigningScheme.ETHSIGN ? SigningScheme.ETHSIGN : SigningScheme.EIP1271
```

Comment:
Are you sure you want 1271 instead of 712?

I don't know which is the right one, just asking